config/templates/cloud-config-controller

```diff
@@ -4,12 +4,35 @@ coreos:
     reboot-strategy: "off"
   flannel:
     interface: $private_ipv4
-    etcd_endpoints: {{ .EtcdEndpoints }}
     etcd_cafile: /etc/kubernetes/ssl/ca.pem
     etcd_certfile: /etc/kubernetes/ssl/etcd-client.pem
     etcd_keyfile: /etc/kubernetes/ssl/etcd-client-key.pem
 
   units:
+    - name: cfn-etcd-environment.service
+      enable: true
+      command: start
+      runtime: true
+      content: |
+        [Unit]
+        Description=Fetches etcd static IP addresses list from CF
+        After=network-online.target
+
+        [Service]
+        Type=oneshot
```

Comment:
You may already know it but according to https://github.com/systemd/systemd/issues/2582, I'd rather like to make this `Type=simple` or implement a loop in bash to retry `cfn-int` on failure for reliability